<!DOCTYPE html><html><head>
      <title>PhosMap Manual</title>
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css">

      
      
      
      
      
      
      
      
      <style>
      code[class*=language-],pre[class*=language-]{color:#333;background:0 0;font-family:Consolas,"Liberation Mono",Menlo,Courier,monospace;text-align:left;white-space:pre;word-spacing:normal;word-break:normal;word-wrap:normal;line-height:1.4;-moz-tab-size:8;-o-tab-size:8;tab-size:8;-webkit-hyphens:none;-moz-hyphens:none;-ms-hyphens:none;hyphens:none}pre[class*=language-]{padding:.8em;overflow:auto;border-radius:3px;background:#f5f5f5}:not(pre)>code[class*=language-]{padding:.1em;border-radius:.3em;white-space:normal;background:#f5f5f5}.token.blockquote,.token.comment{color:#969896}.token.cdata{color:#183691}.token.doctype,.token.macro.property,.token.punctuation,.token.variable{color:#333}.token.builtin,.token.important,.token.keyword,.token.operator,.token.rule{color:#a71d5d}.token.attr-value,.token.regex,.token.string,.token.url{color:#183691}.token.atrule,.token.boolean,.token.code,.token.command,.token.constant,.token.entity,.token.number,.token.property,.token.symbol{color:#0086b3}.token.prolog,.token.selector,.token.tag{color:#63a35c}.token.attr-name,.token.class,.token.class-name,.token.function,.token.id,.token.namespace,.token.pseudo-class,.token.pseudo-element,.token.url-reference .token.variable{color:#795da3}.token.entity{cursor:help}.token.title,.token.title .token.punctuation{font-weight:700;color:#1d3e81}.token.list{color:#ed6a43}.token.inserted{background-color:#eaffea;color:#55a532}.token.deleted{background-color:#ffecec;color:#bd2c00}.token.bold{font-weight:700}.token.italic{font-style:italic}.language-json .token.property{color:#183691}.language-markup .token.tag .token.punctuation{color:#333}.language-css .token.function,code.language-css{color:#0086b3}.language-yaml .token.atrule{color:#63a35c}code.language-yaml{color:#183691}.language-ruby .token.function{color:#333}.language-markdown .token.url{color:#795da3}.language-makefile .token.symbol{color:#795da3}.language-makefile .token.variable{color:#183691}.language-makefile .token.builtin{color:#0086b3}.language-bash .token.keyword{color:#0086b3}pre[data-line]{position:relative;padding:1em 0 1em 3em}pre[data-line] .line-highlight-wrapper{position:absolute;top:0;left:0;background-color:transparent;display:block;width:100%}pre[data-line] .line-highlight{position:absolute;left:0;right:0;padding:inherit 0;margin-top:1em;background:hsla(24,20%,50%,.08);background:linear-gradient(to right,hsla(24,20%,50%,.1) 70%,hsla(24,20%,50%,0));pointer-events:none;line-height:inherit;white-space:pre}pre[data-line] .line-highlight:before,pre[data-line] .line-highlight[data-end]:after{content:attr(data-start);position:absolute;top:.4em;left:.6em;min-width:1em;padding:0 .5em;background-color:hsla(24,20%,50%,.4);color:#f4f1ef;font:bold 65%/1.5 sans-serif;text-align:center;vertical-align:.3em;border-radius:999px;text-shadow:none;box-shadow:0 1px #fff}pre[data-line] .line-highlight[data-end]:after{content:attr(data-end);top:auto;bottom:.4em}html body{font-family:'Helvetica Neue',Helvetica,'Segoe UI',Arial,freesans,sans-serif;font-size:16px;line-height:1.6;color:#333;background-color:#fff;overflow:initial;box-sizing:border-box;word-wrap:break-word}html body>:first-child{margin-top:0}html body h1,html body h2,html body h3,html body h4,html body h5,html body h6{line-height:1.2;margin-top:1em;margin-bottom:16px;color:#000}html body h1{font-size:2.25em;font-weight:300;padding-bottom:.3em}html body h2{font-size:1.75em;font-weight:400;padding-bottom:.3em}html body h3{font-size:1.5em;font-weight:500}html body h4{font-size:1.25em;font-weight:600}html body h5{font-size:1.1em;font-weight:600}html body h6{font-size:1em;font-weight:600}html body h1,html body h2,html body h3,html body h4,html body h5{font-weight:600}html body h5{font-size:1em}html body h6{color:#5c5c5c}html body strong{color:#000}html body del{color:#5c5c5c}html body a:not([href]){color:inherit;text-decoration:none}html body a{color:#08c;text-decoration:none}html body a:hover{color:#00a3f5;text-decoration:none}html body img{max-width:100%}html body>p{margin-top:0;margin-bottom:16px;word-wrap:break-word}html body>ol,html body>ul{margin-bottom:16px}html body ol,html body ul{padding-left:2em}html body ol.no-list,html body ul.no-list{padding:0;list-style-type:none}html body ol ol,html body ol ul,html body ul ol,html body ul ul{margin-top:0;margin-bottom:0}html body li{margin-bottom:0}html body li.task-list-item{list-style:none}html body li>p{margin-top:0;margin-bottom:0}html body .task-list-item-checkbox{margin:0 .2em .25em -1.8em;vertical-align:middle}html body .task-list-item-checkbox:hover{cursor:pointer}html body blockquote{margin:16px 0;font-size:inherit;padding:0 15px;color:#5c5c5c;background-color:#f0f0f0;border-left:4px solid #d6d6d6}html body blockquote>:first-child{margin-top:0}html body blockquote>:last-child{margin-bottom:0}html body hr{height:4px;margin:32px 0;background-color:#d6d6d6;border:0 none}html body table{margin:10px 0 15px 0;border-collapse:collapse;border-spacing:0;display:block;width:100%;overflow:auto;word-break:normal;word-break:keep-all}html body table th{font-weight:700;color:#000}html body table td,html body table th{border:1px solid #d6d6d6;padding:6px 13px}html body dl{padding:0}html body dl dt{padding:0;margin-top:16px;font-size:1em;font-style:italic;font-weight:700}html body dl dd{padding:0 16px;margin-bottom:16px}html body code{font-family:Menlo,Monaco,Consolas,'Courier New',monospace;font-size:.85em;color:#000;background-color:#f0f0f0;border-radius:3px;padding:.2em 0}html body code::after,html body code::before{letter-spacing:-.2em;content:'\00a0'}html body pre>code{padding:0;margin:0;word-break:normal;white-space:pre;background:0 0;border:0}html body .highlight{margin-bottom:16px}html body .highlight pre,html body pre{padding:1em;overflow:auto;line-height:1.45;border:#d6d6d6;border-radius:3px}html body .highlight pre{margin-bottom:0;word-break:normal}html body pre code,html body pre tt{display:inline;max-width:initial;padding:0;margin:0;overflow:initial;line-height:inherit;word-wrap:normal;background-color:transparent;border:0}html body pre code:after,html body pre code:before,html body pre tt:after,html body pre tt:before{content:normal}html body blockquote,html body dl,html body ol,html body p,html body pre,html body ul{margin-top:0;margin-bottom:16px}html body kbd{color:#000;border:1px solid #d6d6d6;border-bottom:2px solid #c7c7c7;padding:2px 4px;background-color:#f0f0f0;border-radius:3px}@media print{html body{background-color:#fff}html body h1,html body h2,html body h3,html body h4,html body h5,html body h6{color:#000;page-break-after:avoid}html body blockquote{color:#5c5c5c}html body pre{page-break-inside:avoid}html body table{display:table}html body img{display:block;max-width:100%;max-height:100%}html body code,html body pre{word-wrap:break-word;white-space:pre}}.markdown-preview{width:100%;height:100%;box-sizing:border-box}.markdown-preview ul{list-style:disc}.markdown-preview ul ul{list-style:circle}.markdown-preview ul ul ul{list-style:square}.markdown-preview ol{list-style:decimal}.markdown-preview ol ol,.markdown-preview ul ol{list-style-type:lower-roman}.markdown-preview ol ol ol,.markdown-preview ol ul ol,.markdown-preview ul ol ol,.markdown-preview ul ul ol{list-style-type:lower-alpha}.markdown-preview .newpage,.markdown-preview .pagebreak{page-break-before:always}.markdown-preview pre.line-numbers{position:relative;padding-left:3.8em;counter-reset:linenumber}.markdown-preview pre.line-numbers>code{position:relative}.markdown-preview pre.line-numbers .line-numbers-rows{position:absolute;pointer-events:none;top:1em;font-size:100%;left:0;width:3em;letter-spacing:-1px;border-right:1px solid #999;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.markdown-preview pre.line-numbers .line-numbers-rows>span{pointer-events:none;display:block;counter-increment:linenumber}.markdown-preview pre.line-numbers .line-numbers-rows>span:before{content:counter(linenumber);color:#999;display:block;padding-right:.8em;text-align:right}.markdown-preview .mathjax-exps .MathJax_Display{text-align:center!important}.markdown-preview:not([data-for=preview]) .code-chunk .code-chunk-btn-group{display:none}.markdown-preview:not([data-for=preview]) .code-chunk .status{display:none}.markdown-preview:not([data-for=preview]) .code-chunk .output-div{margin-bottom:16px}.markdown-preview .md-toc{padding:0}.markdown-preview .md-toc .md-toc-link-wrapper .md-toc-link{display:inline;padding:.25rem 0}.markdown-preview .md-toc .md-toc-link-wrapper .md-toc-link div,.markdown-preview .md-toc .md-toc-link-wrapper .md-toc-link p{display:inline}.markdown-preview .md-toc .md-toc-link-wrapper.highlighted .md-toc-link{font-weight:800}.scrollbar-style::-webkit-scrollbar{width:8px}.scrollbar-style::-webkit-scrollbar-track{border-radius:10px;background-color:transparent}.scrollbar-style::-webkit-scrollbar-thumb{border-radius:5px;background-color:rgba(150,150,150,.66);border:4px solid rgba(150,150,150,.66);background-clip:content-box}html body[for=html-export]:not([data-presentation-mode]){position:relative;width:100%;height:100%;top:0;left:0;margin:0;padding:0;overflow:auto}html body[for=html-export]:not([data-presentation-mode]) .markdown-preview{position:relative;top:0;min-height:100vh}@media screen and (min-width:914px){html body[for=html-export]:not([data-presentation-mode]) .markdown-preview{padding:2em calc(50% - 457px + 2em)}}@media screen and (max-width:914px){html body[for=html-export]:not([data-presentation-mode]) .markdown-preview{padding:2em}}@media screen and (max-width:450px){html body[for=html-export]:not([data-presentation-mode]) .markdown-preview{font-size:14px!important;padding:1em}}@media print{html body[for=html-export]:not([data-presentation-mode]) #sidebar-toc-btn{display:none}}html body[for=html-export]:not([data-presentation-mode]) #sidebar-toc-btn{position:fixed;bottom:8px;left:8px;font-size:28px;cursor:pointer;color:inherit;z-index:99;width:32px;text-align:center;opacity:.4}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] #sidebar-toc-btn{opacity:1}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc{position:fixed;top:0;left:0;width:300px;height:100%;padding:32px 0 48px 0;font-size:14px;box-shadow:0 0 4px rgba(150,150,150,.33);box-sizing:border-box;overflow:auto;background-color:inherit}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc::-webkit-scrollbar{width:8px}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc::-webkit-scrollbar-track{border-radius:10px;background-color:transparent}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc::-webkit-scrollbar-thumb{border-radius:5px;background-color:rgba(150,150,150,.66);border:4px solid rgba(150,150,150,.66);background-clip:content-box}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc a{text-decoration:none}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc .md-toc{padding:0 16px}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc .md-toc .md-toc-link-wrapper .md-toc-link{display:inline;padding:.25rem 0}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc .md-toc .md-toc-link-wrapper .md-toc-link div,html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc .md-toc .md-toc-link-wrapper .md-toc-link p{display:inline}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .md-sidebar-toc .md-toc .md-toc-link-wrapper.highlighted .md-toc-link{font-weight:800}html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .markdown-preview{left:300px;width:calc(100% - 300px);padding:2em calc(50% - 457px - 300px / 2);margin:0;box-sizing:border-box}@media screen and (max-width:1274px){html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .markdown-preview{padding:2em}}@media screen and (max-width:450px){html body[for=html-export]:not([data-presentation-mode])[html-show-sidebar-toc] .markdown-preview{width:100%}}html body[for=html-export]:not([data-presentation-mode]):not([html-show-sidebar-toc]) .markdown-preview{left:50%;transform:translateX(-50%)}html body[for=html-export]:not([data-presentation-mode]):not([html-show-sidebar-toc]) .md-sidebar-toc{display:none}
/* Please visit the URL below for more information: */
/*   https://shd101wyy.github.io/markdown-preview-enhanced/#/customize-css */

      </style>
      <!-- The content below will be included at the end of the <head> element. --><script type="text/javascript">
  document.addEventListener("DOMContentLoaded", function () {
    // your code here
  });
</script></head><body for="html-export">
    
    
      <div class="crossnote markdown-preview  ">
      
<h1 id="procedure">PROCEDURE </h1>
<h2 id="notice">Notice </h2>
<p>The web version of PhosMap is for quick start of visualization only due to the low-level hardware of R shiny server. It is single-threaded and we recommend users to analyze small data sets using the demo server. For larger datasets, upgraded hardware is necessary according to the possible computational cost of the data. We recommend users to use the local docker version of PhosMap.</p>
<h2 id="introduction-of-example-data">Introduction of example data </h2>
<div align="left">
Here, we reanalysis (phospho)proteomic profilings of WiDr colorectal cancer cells harbouring the BRAF(V600E) mutation after treatment using vemurafenibin a time course of 0, 2, 6, 24, and 48 hour<sup>[1]</sup>. The raw files were deposited in ProteomeXchange Consortium(PXD007740).
The raw data were processed in Firmiana, a one-stop proteomic cloud platform<sup>[2]</sup>, to obtain quantitative peptide and protein files. 
You can download example data in <font color="blue">https://github.com/liuzan-info/PhosMap/tree/master/examplefile/mascot</font> and <font color="blue">https://github.com/liuzan-info/PhosMap/tree/master/examplefile/maxquant</font>.     
</div>
<h2 id="preprocessing-for-maxquant-data">Preprocessing for Maxquant data </h2>
<h3 id="import-maxquant-data">Import MaxQuant data </h3>
<h4 id="how-to-import-your-maxquant-data">How to import your MaxQuant data </h4>
<ol>
<li>Go to the ‘Import data’ tab.</li>
<li>Choose 'Maxquant' to start with data from Maxquant.</li>
<li>Click ‘Browse’ to upload phosphoproteomics experimental design file in .txt format, and phospho (STY)Sites.txt. Proteomics experimental design file is optional.</li>
<li>Uploaded data will be shown in the 'Data Overview' secondary tab.</li>
<li>You can also choose ‘load exmaple data’ to use exmaple files.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_maxquant/import_maxquant_data.png" width="100%">
</div>
<h3 id="quality-control-and-merging">Quality control and merging </h3>
<h4 id="function">Function </h4>
<p>Generate merged phosphoproteomics data frame based on peptides files.</p>
<div align="center">
<img src="manual_picture2/qcstep.png" width="50%">
</div>
<div align="center">
<img src="manual_picture2/spc.png" width="25%">
</div>
<h4 id="how-to-get-analysis-results">How to get analysis results </h4>
<ol>
<li>Go to the ‘Preprocessing’ tab.</li>
<li>Modify the parameters in Step1 according to your needs.</li>
<li>Click the running button in Step1 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_maxquant/qc.png" width="100%">
</div>
<h4 id="parameter-selection-explanation">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>Minimum Score</strong>: The minimum score of credible peptides. Default is 40; adjust higher for more confidence or lower to retain more data.</p>
</li>
<li>
<p><strong>Minimum Localization Probability</strong>: Determines the confidence level for modification site accuracy, typically set at 0.75, indicating 75% confidence.</p>
</li>
<li>
<p><strong>Minimum Detection Frequency</strong>: Minimum detection frequency for per locus, equivalents to the number of samples minus the number of '0' value.</p>
</li>
</ul>
<h4 id="interpretation-of-analysis-results">Interpretation of analysis results </h4>
<p>We performed quality control for identified phosphopeptides using PhosMap, those phosphopeptides that met 1% FDR at peptide level and had ion score greater than 20 and the highest confidence probabilities of p-sites computed by Mascot, were kept. We merged phosphopeptides list with quantitative value from all experiments to generate a matrix for analysis.</p>
<h3 id="data-normalization">Data normalization </h3>
<h4 id="function-1">Function </h4>
<p>PhosMap provides two kinds of normalizations, a total sum scaling normalization and normalizing phosphoproteomics data based on proteomics data.</p>
<div align="center">
<img src="manual_picture2/normalizationstep.png" width="50%">
</div>
<h4 id="how-to-get-analysis-results-1">How to get analysis results </h4>
<ol>
<li>Go to the ‘Preprocessing’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the running button in the Step2 and the normalized data of p-sites based on a total sum scaling will apear on the right.</li>
<li>Click the running button in Step3 and normalized data of p-sites based on proteomics data will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_maxquant/norm1.png" width="100%">
</div>
<div align="center">
<img src="manual_picture2/preprocessing_maxquant/norm2.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-1">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>Normalization Method</strong>: The approach used to address unequal loading of phosphopeptides. 'Global' or 'median' refer to scaling the intensities of phosphopeptides globally or based on the median value.</p>
</li>
<li>
<p><strong>Imputation Method</strong>: Strategies for replacing missing values. 'Globally' or 'by group' refer to imputing missing values across the entire dataset or within experimental groups.</p>
<ul>
<li>'0': Assigns a value of zero to missing data points.</li>
<li>'Minimum': Uses the smallest value in the dataset for imputation.</li>
<li>'Minimum/10': Imputes missing values with the smallest value divided by 10.</li>
<li>'BPCA': Utilizes Bayesian Principal Component Analysis.</li>
<li>'LLS': Employs Local Least Squares.</li>
<li>'KNNMethod': Applies k-Nearest Neighbors for imputation.</li>
<li>'RowMedian': Replaces missing data with the row's median value.</li>
<li>'ImpSeq': Sequential imputation method.</li>
<li>'ImpSeqProb': Probabilistic approach to imputation.</li>
<li>'ColMedian': Fills in missing values with the column's median.</li>
</ul>
</li>
<li>
<p><strong>Top</strong>: Compute row maximum each psites, sort row maximum in decreasing order and keep top N (percentage). The more this value, the more phosphopeptides are included.</p>
</li>
<li>
<p><strong>With Proteomics Data</strong>: This option should be selected if you need normalization with protein profiling data.</p>
</li>
<li>
<p><strong>Intensity Type</strong>: Users have options: 'Intensity' for raw abundance measurements, 'iBAQ' for intensity-Based Absolute Quantification which estimates absolute protein amounts, and 'LFQ intensity' for Label-Free Quantification, allowing comparison of protein abundances across samples without labels.</p>
</li>
<li>
<p><strong>Minimum Unique Peptide</strong>: Sets the minimum number of unique peptides that must be detected for a protein's quantitation to be considered reliable. A minimum of 1 means even proteins identified by a single unique peptide are included.</p>
</li>
</ul>
<h2 id="preprocessing-for-firmiana-data">Preprocessing for Firmiana data </h2>
<h3 id="import-firmiana-data">Import Firmiana data </h3>
<h4 id="how-to-import-your-firmiana-data">How to import your Firmiana data </h4>
<ol>
<li>Go to the ‘Import data’ tab.</li>
<li>Choose 'Firmiana' to start with data from Firmiana.</li>
<li>Click ‘Browse’ to upload phosphoproteomics experimental design file in .txt format.</li>
<li>Zip your Mascot xml files and Phosphoproteomics peptide files, and then upload. The folder tree is shown below. File names of Mascot xml files and Phosphoproteomics peptide files must be consistent with ’Experiment_Code’ of phosphoproteomics experimental design file.</li>
</ol>
<div align="center">
<img src="manual_picture2/tree1.png" width="25%">
<img src="manual_picture2/phosphorylation_exp_design_info_csv.png" width="50%">
</div>
<ol start="6">
<li>Proteomics data is optional. Click ‘Browse’ to upload proteomics experimental design file in .txt format. Zip your Profiling_gene_txt and upload. The folder tree is shown below. File names of Profiling_gene_txt must be consistent with ‘Expriment_Code’ of proteomics experimental design file.</li>
</ol>
<div align="center">
<img src="manual_picture2/tree2.png" width="25%">
<img src="manual_picture2/profiling_exp_design_info_csv.png" width="50%">
</div>
<ol start="7">
<li>Uploaded data will be shown in the 'Data Overview' secondary tab.</li>
<li>You can also choose ‘load exmaple data’ to use exmaple files.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/import_firmiana_data.png" width="100%">
</div>
<h3 id="parser">Parser </h3>
<h4 id="function-2">Function </h4>
<p>If you start with .xml files from mascot results, you can run this button to parser them to sites score files, based on which .csv files of phosphorylation sites with confidence score will be genereated.</p>
<div align="center">
<img src="manual_picture2/parserstep.png" width="25%">
</div>
<h4 id="how-to-get-analysis-results-2">How to get analysis results </h4>
<ol>
<li>Go to the 'Preprocessing' tab.</li>
<li>Click the running button in Step1 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/parser.png" width="100%">
</div>
<h3 id="quality-control-and-merging-1">Quality control and merging </h3>
<h4 id="function-3">Function </h4>
<p>Generate merged phosphoproteomics data frame based on peptides files.</p>
<div align="center">
<img src="manual_picture2/qcstep.png" width="50%">
</div>
<div align="center">
<img src="manual_picture2/spc.png" width="25%">
</div>
<h4 id="how-to-get-analysis-results-3">How to get analysis results </h4>
<ol>
<li>Go to the ‘Preprocessing’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the running button in Step2 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/qc.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-2">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>Minimum Score</strong>: The minimum score of credible peptides. Default is 20; adjust higher for more confidence or lower to retain more data.</p>
</li>
<li>
<p><strong>Minimum FDR</strong>: The minimum FDR of credible peptides. The default FDR threshold is 0.01, which means that only 1% of the identified peptides may be false positives. Users should set this according to the level of specificity and sensitivity required for their analysis.</p>
</li>
</ul>
<h4 id="interpretation-of-analysis-results-1">Interpretation of analysis results </h4>
<p>We performed quality control for identified phosphopeptides using PhosMap, those phosphopeptides that met 1% FDR at peptide level and had ion score greater than 20 and the highest confidence probabilities of p-sites computed by Mascot, were kept. We merged phosphopeptides list with quantitative value from all experiments to generate a matrix for analysis.</p>
<h3 id="mapping-p-sites-to-protein">Mapping p-sites to protein </h3>
<h4 id="function-4">Function </h4>
<p>Mapping protein gi number to gene symbol and outputing expression profile matrix with gene symbol.<br>
Constructing the data frame with unique phosphorylation site for each protein sequence.</p>
<div align="center">
<img src="manual_picture2/mappingstep.png" width="50%">
</div>
<h4 id="how-to-get-analysis-results-4">How to get analysis results </h4>
<ol>
<li>Go to the ‘Preprocessing’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the running button in Step3 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/mapping.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-3">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>Species</strong>: Users select 'human', 'mouse' or 'rattus' to map peptide sequences against the human protein database.</p>
</li>
<li>
<p><strong>ID type</strong>: Offers multiple identifier options for proteins:</p>
<ul>
<li>'RefSeq_Protein_GI': Uses NCBI's Reference Sequence (RefSeq) GenInfo Identifier numbers. Example: GI 4502027.</li>
<li>'RefSeq_Protein_Accession': Utilizes the RefSeq accession numbers. Example: NP_001101.1.</li>
<li>'Uniprot_Protein_Accession': Employs accession numbers from the Universal Protein Resource (UniProt). Example: P60709.</li>
<li>'GeneID': Refers to the numeric gene identifier used by NCBI. Example: ACTB.</li>
</ul>
</li>
<li>
<p><strong>fasta type</strong>: Indicates the database format for protein sequences. Options are 'refseq' for NCBI's RefSeq database or 'uniprot' for the UniProt database. Selection should align with the user's sequence data and the ID type chosen to ensure consistency in mapping results.</p>
</li>
</ul>
<h4 id="interpretation-of-analysis-results-2">Interpretation of analysis results </h4>
<p>Combining the phosphopeptides sequence, modification position, attached protein ID and the built-in human protein reference database of PhosMap, all p-sites were mapped to the corresponding protein sequence and represented by unique p-sites identifier (upsID) that consisted of a protein GI number/accession, gene symbol and location of the p-site in the protein sequence. In addition, the matched proteome data with phosphoproteome were collected at each time point in Ressa, et al. study. Finally, 3,649 unique p-sites were obtained and their quantitative values were normalized by matched protein profiling data using PhosMap.</p>
<h3 id="data-normalization-1">Data normalization </h3>
<h4 id="function-5">Function </h4>
<p>PhosMap provides two kinds of normalizations, a total sum scaling normalization and normalizing phosphoproteomics data based on proteomics data.</p>
<div align="center">
<img src="manual_picture2/normalizationstep.png" width="50%">
</div>
<h4 id="how-to-get-analysis-results-5">How to get analysis results </h4>
<ol>
<li>Go to the ‘Preprocessing’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the running button in the Step4 and the normalized data of p-sites based on a total sum scaling will apear on the right.</li>
<li>Click the running button in Step5 and normalized data of p-sites based on proteomics data will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/norm1.png" width="100%">
</div>
<div align="center">
<img src="manual_picture2/preprocessing_firmiana/norm2.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-4">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>Minimum Detection Frequency</strong>: Minimum detection frequency for per locus, equivalents to the number of samples minus the number of '0' value.</p>
</li>
<li>
<p><strong>Normalization Method</strong>: The approach used to address unequal loading of phosphopeptides. 'Global' or 'median' refer to scaling the intensities of phosphopeptides globally or based on the median value.</p>
</li>
<li>
<p><strong>Imputation Method</strong>: Strategies for replacing missing values. 'Globally' or 'by group' refer to imputing missing values across the entire dataset or within experimental groups.</p>
<ul>
<li>'0': Assigns a value of zero to missing data points.</li>
<li>'Minimum': Uses the smallest value in the dataset for imputation.</li>
<li>'Minimum/10': Imputes missing values with the smallest value divided by 10.</li>
<li>'BPCA': Utilizes Bayesian Principal Component Analysis.</li>
<li>'LLS': Employs Local Least Squares.</li>
<li>'KNNMethod': Applies k-Nearest Neighbors for imputation.</li>
<li>'RowMedian': Replaces missing data with the row's median value.</li>
<li>'ImpSeq': Sequential imputation method.</li>
<li>'ImpSeqProb': Probabilistic approach to imputation.</li>
<li>'ColMedian': Fills in missing values with the column's median.</li>
</ul>
</li>
<li>
<p><strong>Top</strong>: Compute row maximum each psites, sort row maximum in decreasing order and keep top N (percentage). The more this value, the more phosphopeptides are included.</p>
</li>
<li>
<p><strong>Control Label</strong>: Allows selection of a reference label for comparative analysis.</p>
</li>
<li>
<p><strong>US Cutoff</strong>: A user-set threshold for inclusion based on abundance or statistical significance.</p>
</li>
</ul>
<h2 id="preprocessing-for-spectronaut-data">Preprocessing for Spectronaut data </h2>
<h3 id="import-spectronaut-data">Import Spectronaut data </h3>
<h4 id="how-to-import-your-spectronaut-data">How to import your Spectronaut data </h4>
<ol>
<li>Go to the ‘Import data’ tab.</li>
<li>Choose 'Spectronaut' under 'DIA' to start with data from Spectronaut.</li>
<li>Click ‘Browse’ to upload phosphoproteomics experimental design file in .txt format.</li>
<li>Click ‘Browse’ to upload Report file generated by Spectronaut in xls format.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_spectronaut/import_spectronaut_data.png" width="100%">
</div>
<h3 id="parser--p-site-quality-control">Parser &amp; p-site Quality Control </h3>
<h4 id="function-6">Function </h4>
<p>If you start with .xls files from Spectronaut results, you can run this button to parser them to sites score files, based on which .csv files of phosphorylation sites with confidence score will be genereated.</p>
<div align="center">
<img src="manual_picture2/parserstep.png" width="25%">
</div>
<h4 id="how-to-get-analysis-results-6">How to get analysis results </h4>
<ol>
<li>Go to the 'Preprocessing' tab.</li>
<li>Click the running button in Step1 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_spectronaut/parser.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-5">Parameter Selection Explanation </h4>
<ul>
<li><strong>Minimum Detection Frequency</strong>: Minimum detection frequency for per locus, equivalents to the number of samples minus the number of '0' value.</li>
</ul>
<h3 id="normalizaiton--imputation--filtering">Normalizaiton &amp; Imputation &amp; Filtering </h3>
<h4 id="function-7">Function </h4>
<p>PhosMap performs a total sum scaling normalization and imputation for missing values with various methods.</p>
<div align="center">
<img src="manual_picture2/normalizationstep.png" width="50%">
</div>
<h4 id="how-to-get-analysis-results-7">How to get analysis results </h4>
<ol>
<li>Go to the 'Preprocessing' tab.</li>
<li>Click the running button in Step2 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_spectronaut/norm.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-6">Parameter Selection Explanation </h4>
<p><strong>Normalization Method</strong>: The approach used to address unequal loading of phosphopeptides. 'Global' or 'median' refer to scaling the intensities of phosphopeptides globally or based on the median value.</p>
<p><strong>Imputation Method</strong>: Strategies for replacing missing values. 'Globally' or 'by group' refer to imputing missing values across the entire dataset or within experimental groups.</p>
<ul>
<li>'0': Assigns a value of zero to missing data points.</li>
<li>'Minimum': Uses the smallest value in the dataset for imputation.</li>
<li>'Minimum/10': Imputes missing values with the smallest value divided by 10.</li>
<li>'BPCA': Utilizes Bayesian Principal Component Analysis.</li>
<li>'LLS': Employs Local Least Squares.</li>
<li>'KNNMethod': Applies k-Nearest Neighbors for imputation.</li>
<li>'RowMedian': Replaces missing data with the row's median value.</li>
<li>'ImpSeq': Sequential imputation method.</li>
<li>'ImpSeqProb': Probabilistic approach to imputation.</li>
<li>'ColMedian': Fills in missing values with the column's median.<br>
<strong>top</strong>: Compute row maximum each psites, sort row maximum in decreasing order and keep top N (percentage). The more this value, the more phosphopeptides are included.</li>
</ul>
<h2 id="preprocessing-for-dia-nn-data">Preprocessing for Dia-NN data </h2>
<h3 id="import-dia-nn-data">Import Dia-NN data </h3>
<h4 id="how-to-import-your-dia-nn-data">How to import your Dia-NN data </h4>
<ol>
<li>Go to the ‘Import data’ tab.</li>
<li>Choose 'Dia-NN' under 'DIA' to start with data from Dia-NN.</li>
<li>Click ‘Browse’ to upload phosphoproteomics experimental design file in .txt format.</li>
<li>Click ‘Browse’ to upload Report file generated by Dia-NN in tsv format.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_diann/import_diann_data.png" width="100%">
</div>
<h3 id="parser--p-site-quality-control-1">Parser &amp; p-site Quality Control </h3>
<h4 id="function-8">Function </h4>
<p>If you start with .tsv files from Dia-NN results, you can run this button to parser them to sites score files, based on which .csv files of phosphorylation sites with confidence score will be genereated.</p>
<div align="center">
<img src="manual_picture2/parserstep.png" width="25%">
</div>
<h4 id="how-to-get-analysis-results-8">How to get analysis results </h4>
<ol>
<li>Go to the 'Preprocessing' tab.</li>
<li>Click the running button in Step1 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_diann/parser.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-7">Parameter Selection Explanation </h4>
<ul>
<li><strong>Minimum Detection Frequency</strong>: Minimum detection frequency for per locus, equivalents to the number of samples minus the number of '0' value.</li>
<li><strong>PTM.Q.Value threshold</strong>: The minimum Q value of credible peptides. Default is 0.01; adjust higher for more confidence or lower to retain more data.</li>
<li><strong>Species</strong>: Users select 'human', 'mouse' or 'rattus' to map peptide sequences against the human protein database.</li>
</ul>
<h3 id="normalizaiton--imputation--filtering-1">Normalizaiton &amp; Imputation &amp; Filtering </h3>
<h4 id="function-9">Function </h4>
<p>PhosMap performs a total sum scaling normalization and imputation for missing values with various methods.</p>
<div align="center">
<img src="manual_picture2/normalizationstep.png" width="50%">
</div>
<h4 id="how-to-get-analysis-results-9">How to get analysis results </h4>
<ol>
<li>Go to the 'Preprocessing' tab.</li>
<li>Click the running button in Step2 and the file will apear on the right.</li>
</ol>
<div align="center">
<img src="manual_picture2/preprocessing_diann/norm.png" width="100%">
</div>
<h4 id="parameter-selection-explanation-8">Parameter Selection Explanation </h4>
<p><strong>Normalization Method</strong>: The approach used to address unequal loading of phosphopeptides. 'Global' or 'median' refer to scaling the intensities of phosphopeptides globally or based on the median value.<br>
<strong>Imputation Method</strong>: Strategies for replacing missing values. 'Globally' or 'by group' refer to imputing missing values across the entire dataset or within experimental groups.</p>
<ul>
<li>'0': Assigns a value of zero to missing data points.</li>
<li>'Minimum': Uses the smallest value in the dataset for imputation.</li>
<li>'Minimum/10': Imputes missing values with the smallest value divided by 10.</li>
<li>'BPCA': Utilizes Bayesian Principal Component Analysis.</li>
<li>'LLS': Employs Local Least Squares.</li>
<li>'KNNMethod': Applies k-Nearest Neighbors for imputation.</li>
<li>'RowMedian': Replaces missing data with the row's median value.</li>
<li>'ImpSeq': Sequential imputation method.</li>
<li>'ImpSeqProb': Probabilistic approach to imputation.</li>
<li>'ColMedian': Fills in missing values with the column's median.<br>
<strong>top</strong>: Compute row maximum each psites, sort row maximum in decreasing order and keep top N (percentage). The more this value, the more phosphopeptides are included.</li>
</ul>
<h2 id="analysis-and-visualization">Analysis and visualization </h2>
<p>PhosMap incorporated six analysis modules: dimension reduction analysis, differential expression analysis, time course analysis, kinase activity prediction, phosphorylation motif enrichment analysis and survival analysis.</p>
<h3 id="upload-data">Upload data </h3>
<h4 id="function-10">Function </h4>
<p>In this step, you can upload your preprocessed data to PhosMap, such as the phosphorylation dataframe. If you have not preprocessed your data, you must preprocess it with PhosMap (go to the ‘Preprocessing’ tab) or do it yourself.</p>
<h4 id="how-to-upload-your-data">How to upload your data </h4>
<ol>
<li>Go to the 'Upload data' under 'Analysis' tab.</li>
<li>Choose 'Load example data' or follow the prompts to upload your own corresponding four files. If you have not preprocessed your data, you can click 'Go to preprocessing' to preprocess it.</li>
</ol>
<div align="center">
<img src="manual_picture2/analysis/import_data.png" width="100%">
</div>
<h3 id="dimension-reduction-analysis">Dimension reduction analysis </h3>
<h4 id="function-11">Function </h4>
<p>In PhosMap, Dimension reduction analysis methods allowed for PCA, t-SNE and UMAP.</p>
<h4 id="the-meaning-of-the-parameters">The meaning of the parameters </h4>
<ol>
<li>‘Title’ refers to the main title of the plot.</li>
<li>‘Legend title’ refers to the title of the legend in the plot.</li>
<li>‘Random seed’ is a parameter for t-SNE that sets the seed for the random number generator. This can be used to ensure reproducibility of results.</li>
<li>‘Perplexity’ is a numerical value for t-SNE, with a default value of 2. It balances the focus between preserving the local and global structure of the data.</li>
<li>‘Neighbors’ is a parameter for UMAP that refers to the size of the local neighborhood (in terms of the number of neighboring sample points) used for manifold approximation. Larger values result in more global views of the manifold, while smaller values result in more local data being preserved.</li>
</ol>
<h4 id="how-to-get-analysis-results-10">How to get analysis results </h4>
<ol>
<li>Go to the 'Dimension reduction analysis' under ‘Analysis’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the ‘Analysis’ button.</li>
<li>The PCA, t-SNE and UMAP plot after running will appear on the right.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/dimensionreductionanalysis.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-9">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>PCA (Principal Component Analysis)</strong>:</p>
<ul>
<li><strong>main</strong>: The title of result plot.</li>
<li><strong>legend title</strong>: The title of result plot legend. 'Hour' can be used for time series.</li>
<li><strong>group mean</strong>: When selected, PCA calculates the average of each group.</li>
</ul>
</li>
<li>
<p><strong>t-SNE</strong>:</p>
<ul>
<li><strong>main</strong>: The title of result plot.</li>
<li><strong>legend title</strong>: The title of result plot legend. 'Hour' can be used for time series.</li>
<li><strong>random seed</strong>: A set value for consistent t-SNE outcomes. If you need reproducible results, set the same value for each run.</li>
<li><strong>perplexity</strong>: Influences the focus on local vs. global data relationships. A low value is for detailed local patterns, higher values for broader data trends.</li>
</ul>
</li>
<li>
<p><strong>UMAP (Uniform Manifold Approximation and Projection)</strong>:</p>
<ul>
<li><strong>main</strong>: The title of result plot.</li>
<li><strong>legend title</strong>: The title of result plot legend. 'Hour' can be used for time series.</li>
<li><strong>neighbors</strong>: Affects the algorithm's emphasis on local/global data features. '5' offers a general view, adjust based on desired data granularity.</li>
</ul>
</li>
</ul>
<h4 id="interpretation-of-analysis-results-3">Interpretation of analysis results </h4>
<p>To extract an overview of the effect of the different time course treatments, we performed PCA analysis in the downstream analysis module of PhosMap. We could see that phosphorylation expression profiles of colorectal cancer cells after longer (24h and 48h) vemurafenibin treatment were quite different from those after short treatment (2h and 6h). In addition, it shows that principal component 1 (PC1), with 31.77%, is superior to 20% from original literature and demonstrates phosphorylation expression profile normalized by matched proteomics data has an advantage over representing the variation over time in the BRAFi-treated samples.</p>
<h3 id="differential-expression-analysis">Differential expression analysis </h3>
<h4 id="function-12">Function </h4>
<p>In PhosMap, differential expression analysis methods allowed for limma, SAM and ANOVA Data analysis.</p>
<h4 id="the-meaning-of-the-parameters-1">The meaning of the parameters </h4>
<ol>
<li>
<p>‘Control’ refers to the control group in the experiment.</p>
</li>
<li>
<p>‘Experiment’ refers to the experimental group in the experiment.</p>
</li>
<li>
<p>‘P-value threshold’ is the threshold for determining statistical significance based on the p-value.</p>
</li>
<li>
<p>‘P-value adjust method’ is the method used to adjust p-values for multiple comparisons.</p>
</li>
<li>
<p>‘FC threshold’ is the fold change threshold for determining significant changes in phosphorylation levels.</p>
</li>
<li>
<p>‘nperms’ is a parameter for the SAM method that specifies the number of permutations to perform.</p>
</li>
<li>
<p>‘Minimum FDR’ is the minimum false discovery rate threshold for determining statistical significance.</p>
</li>
<li>
<p>‘Clustering distance rows’ is a parameter for heatmap generation that specifies the distance metric used for clustering rows.</p>
</li>
<li>
<p>‘Clustering method’ is a parameter for heatmap generation that specifies the clustering method used to cluster rows and columns.</p>
</li>
</ol>
<h4 id="how-to-get-analysis-results-11">How to get analysis results </h4>
<ol>
<li>Go to the 'Differential Expression Analysis' under ‘Analysis’ tab.</li>
<li>Go to the 'limma', 'SAM' or 'ANOVA' secondary tab.</li>
<li>Choose Control and Experiment used for differential Expression Analysis.</li>
<li>Choose 'Interactive mode' and click the 'Analysis’ button. The interactive plot after running will appear on the right.</li>
<li>Choose 'Static mode' and click the 'Analysis’ button. The static plot after running will appear on the right.</li>
<li>Click ‘Plot Heatmap’ button. The heatmap will apear in the pop-up window.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/limma1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/limma2.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/limma3.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/sam1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/sam2.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/anova1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/anova2.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-10">Parameter Selection Explanation </h4>
<ul>
<li>
<p><strong>limma</strong>:</p>
<ul>
<li>
<p>Control: Identifies the control group in the experiment. Choose based on the group that represents the normal or untreated condition.</p>
</li>
<li>
<p>Experiment: Specifies the group subjected to the experimental condition or treatment.</p>
</li>
<li>
<p>p-value Threshold: Sets the cutoff for statistical significance, typically at 0.05. Lower this threshold for stricter criteria.</p>
</li>
<li>
<p>p-value Adjust Method: Selects the correction method for multiple comparisons. Available methods include:</p>
<ul>
<li>'none': No adjustment for multiple comparisons.</li>
<li>'holm': Holm's sequential Bonferroni method.</li>
<li>'hochberg': Hochberg's step-up procedure.</li>
<li>'hommel': Hommel's correction.</li>
<li>'bonferroni': Bonferroni correction, very stringent, increases with the number of tests.</li>
<li>'BH': Benjamini-Hochberg method, controls the false discovery rate.</li>
<li>'BY': Benjamini-Yekutieli method, suitable for when tests are dependent.</li>
<li>'fdr': A general term for any method that controls the false discovery rate.</li>
</ul>
</li>
<li>
<p>FC Threshold: Establishes the minimum fold change for a p-site to be considered significantly different, commonly set at 2.</p>
</li>
<li>
<p>Title: Title of the plot.</p>
</li>
<li>
<p>X Axis Label: Denotes the label for the X-axis, usually 'log2FC' for fold change.</p>
</li>
<li>
<p>Y Axis Label: Denotes the label for the Y-axis, often '-log10(p-value)' to represent statistical significance.</p>
</li>
<li>
<p>"UP" Colour: The color representing upregulated genes.</p>
</li>
<li>
<p>"DOWN" Colour: The color indicating downregulated genes.</p>
</li>
<li>
<p>"NOT" Colour: The color for genes without significant change.</p>
</li>
</ul>
</li>
<li>
<p><strong>SAM</strong>:</p>
<ul>
<li>Control: Identifies the control group in the experiment. Choose based on the group that represents the normal or untreated condition.</li>
<li>Experiment: Specifies the group subjected to the experimental condition or treatment.</li>
<li>nperms: Number of permutations to perform during the SAM analysis. A higher number of permutations can provide a more accurate estimation of the false discovery rate but will increase computation time.</li>
<li>Minimum FDR: The threshold for the False Discovery Rate, the expected proportion of false positives among the declared significant tests. A common threshold is 0.05, indicating a 5% expected rate of false discoveries.</li>
</ul>
</li>
<li>
<p><strong>ANOVA</strong>:</p>
<ul>
<li>FC threshold: Establishes the minimum fold change for a p-site to be considered significantly different, commonly set at 2.</li>
<li>p-value Threshold: Sets the cutoff for statistical significance, typically at 0.05. Lower this threshold for stricter criteria.</li>
<li>p-value adjust method: Selects the correction method for multiple comparisons. Available methods include:
<ul>
<li>'none': No adjustment for multiple comparisons.</li>
<li>'holm': Holm's sequential Bonferroni method.</li>
<li>'hochberg': Hochberg's step-up procedure.</li>
<li>'hommel': Hommel's correction.</li>
<li>'bonferroni': Bonferroni correction, very stringent, increases with the number of tests.</li>
<li>'BH': Benjamini-Hochberg method, controls the false discovery rate.</li>
<li>'BY': Benjamini-Yekutieli method, suitable for when tests are dependent.</li>
<li>'fdr': A general term for any method that controls the false discovery rate.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>Heatmap</strong>:</p>
<ul>
<li>
<p>Scale: Adjusts the data for heatmap visualization, affecting how patterns and gradients are displayed. Options include:</p>
<ul>
<li>'none': No scaling is performed, displaying the raw values.</li>
<li>'row': Normalizes the data within each row.</li>
<li>'column': Normalizes the data within each column.</li>
</ul>
</li>
<li>
<p>Cluster by Row: If selected, rows will be grouped based on similarity, which is useful for identifying patterns in data.</p>
</li>
<li>
<p>Clustering Distance Rows: Selects the metric used to calculate the distance between rows in the heatmap, which affects how rows are clustered based on their similarity.</p>
<ul>
<li>'euclidean': A standard distance measure that calculates the root of square differences.</li>
<li>'correlation': Measures the degree to which rows are correlated. This method is useful when the pattern of change is more important than the magnitude of change.</li>
</ul>
</li>
<li>
<p>Clustering Method: The algorithm used to cluster rows or columns in the heatmap, which influences how the patterns of similarity are grouped.</p>
<ul>
<li>'ward.D2': Minimizes the sum of squared differences within all clusters.</li>
<li>'ward.D': An alternative Ward's method.</li>
<li>'single': Uses the minimum of the distances between all observations of the two sets.</li>
<li>'complete': Uses the maximum distances between all observations of the two sets.</li>
<li>'average': Uses the average of the distances of each observation of the two sets.</li>
<li>'mcquitty': Similar to average, but uses a linkage function that is a variant of the WPGMA method (weighted pair group method with averaging).</li>
<li>'median': Uses the median of the distances between all observations of the two sets.</li>
<li>'centroid': Uses the centroid of the clusters.</li>
</ul>
</li>
<li>
<p>Miss Row Name: If selected, indicates that rownames are not displayed in the heatmap.</p>
</li>
</ul>
</li>
</ul>
<h4 id="interpretation-of-analysis-results-4">Interpretation of analysis results </h4>
<p>In order to show differential expression analysis between two experimental conditions. We use the limma method integrated into differential expression analysis module of PhosMap to identify 128 significant differently expressed p-sites (DEPs) between the samples with BRAFi-treated for two hours and control samples (P value &lt; 0.05 and fold change &gt; 2). 139 p-sites were up-regulated in the BRAFi-treated samples. The most disparate difference is observed in DAP_S51, whose phosphoserine is related to the MTOR pathway. 99 p-sites were down-regulated in the BRAFi-treated samples.</p>
<p>For the multiple experimental conditions, we leveraged the embedded ANOVA analysis of PhosMap and identified 548 DEPs among the five time points (P value &lt; 0.1 and fold change &gt; 2).</p>
<h3 id="time-course-analysis">Time Course Analysis </h3>
<h4 id="function-13">Function </h4>
<p>Fuzzy clustering was applied to time course analysis for discovering patterns associated with time points in PhosMap.The corresponding line chart combined with membership for each cluster was also drawn.</p>
<h4 id="the-meaning-of-the-parameters-2">The meaning of the parameters </h4>
<ol>
<li>‘Minimum membership value’ is a threshold for determining the minimum membership value for a data point to be included in a cluster.</li>
<li>‘Iteration’ is the number of iterations to perform in the clustering algorithm.</li>
<li>‘Number of clusters’ is the number of clusters to generate in the clustering algorithm.</li>
</ol>
<h4 id="how-to-get-analysis-results-12">How to get analysis results </h4>
<ol>
<li>Go to the ‘Time course Analysis (fuzzy clustering)’ under ‘Analysis’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the ‘Analysis’ button. The plot after running will appear on the right.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<h4 id="interpretation-of-analysis-results-5">Interpretation of analysis results </h4>
<p>These 548 DEPs were used as inputs in the time course analysis module of PhosMap, then 9 strong expression patterns were generated. Two major clusters show significant downregulation at the phosphoproteomics signalling level upon BRAFi treatment in line with the original literature. Cluster 1 responds within 2 hours, an early treatment response. Cluster 2 responds within 24 hours, a late treatment response.</p>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/timecourse.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-11">Parameter Selection Explanation </h4>
<ul>
<li><strong>p-value Threshold</strong>: Sets the cutoff for statistical significance, typically at 0.05. Lower this threshold for stricter criteria.</li>
<li><strong>p-value Adjust Method</strong>: Selects the correction method for multiple comparisons. Available methods include:
<ul>
<li>'none': No adjustment for multiple comparisons.</li>
<li>'holm': Holm's sequential Bonferroni method.</li>
<li>'hochberg': Hochberg's step-up procedure.</li>
<li>'hommel': Hommel's correction.</li>
<li>'bonferroni': Bonferroni correction, very stringent, increases with the number of tests.</li>
<li>'BH': Benjamini-Hochberg method, controls the false discovery rate.</li>
<li>'BY': Benjamini-Yekutieli method, suitable for when tests are dependent.</li>
<li>'fdr': A general term for any method that controls the false discovery rate.</li>
</ul>
</li>
<li><strong>FC threshold</strong>: Establishes the minimum fold change for a p-site to be considered significantly different, commonly set at 2.</li>
<li><strong>Minimum Membership Value</strong>: Determines how strictly data points are assigned to clusters in fuzzy clustering. A higher value means a data point must have a stronger association with a cluster to be included, making clusters more distinct. A lower value allows for more overlap between clusters, which may reflect subtle gradations in data.</li>
<li><strong>Iteration</strong>: Specifies how many times the clustering algorithm will be run to refine the clusters. More iterations can lead to more accurate clustering but will take longer to compute. Fewer iterations will be quicker but may result in less precise clustering.</li>
<li><strong>Number of Clusters</strong>: The predefined number of clusters to divide the data into.</li>
</ul>
<h3 id="kinase-activity-prediction-ksea">Kinase activity prediction (KSEA) </h3>
<h4 id="function-14">Function </h4>
<p>In PhosMap, KSEA was used to predict kinase activity.</p>
<h4 id="the-meaning-of-the-parameters-3">The meaning of the parameters </h4>
<ol>
<li>‘Control’ refers to the control group in the experiment.</li>
<li>‘Experiment’ refers to the experimental group in the experiment.</li>
<li>‘Species’ refers to the species of the organism being studied.</li>
<li>‘Scale’ is a parameter for scaling the data before generating the heatmap.</li>
<li>‘Clustering distance rows’ is a parameter for heatmap generation that specifies the distance metric used for clustering rows.</li>
<li>‘Clustering method’ is a parameter for heatmap generation that specifies the clustering method used to cluster rows and columns.</li>
</ol>
<h4 id="how-to-get-analysis-results-13">How to get analysis results </h4>
<ol>
<li>Go to the ‘Kinase-Substrate Enrichment Analysis’ under ‘Analysis’ tab.</li>
<li>Select ‘Multiple groups’ or ‘Two groups’ according to the number of groups of your data.</li>
<li>Click the first 'Analysis' button. If ‘Multiple groups’ is selected, after running, the plot will appear on the right. Click ‘view result’ to view and download the kinase prediction time course result. If ‘Two groups’ is selected, only the phoshorylation dataframe will appear on the right.</li>
<li>Select a cluster if ‘Multiple groups’ is selected. Click the second ‘Analysis’ button. After running, the heatmap will appear on the right.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/kap1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/kap2.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/kap3.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/kap4.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-12">Parameter Selection Explanation </h4>
<ul>
<li><strong>Select a Cluster</strong>: Determines which cluster from previous time course analyses to use for enrichment analysis. Different clusters may yield different insights into kinase-substrate relationships.</li>
<li><strong>Species</strong>: Users select 'human', 'mouse' or 'rattus' to map peptide sequences against the human protein database.</li>
<li><strong>Scale</strong>: Influences the representation of data. Without scaling ('none'), the raw data is analyzed; if scaled, differences in expression levels may be emphasized ('row') or standardized across samples ('column').</li>
<li><strong>Clustering Distance Rows</strong>: The metric for measuring distances between data points in clustering. 'Euclidean' distance measures actual distances, whereas a metric like 'correlation' would consider similarity in patterns of expression.</li>
<li><strong>Clustering Method</strong>: The algorithm used to cluster rows or columns in the heatmap, which influences how the patterns of similarity are grouped.
<ul>
<li>'ward.D2': Minimizes the sum of squared differences within all clusters.</li>
<li>'ward.D': An alternative Ward's method.</li>
<li>'single': Uses the minimum of the distances between all observations of the two sets.</li>
<li>'complete': Uses the maximum distances between all observations of the two sets.</li>
<li>'average': Uses the average of the distances of each observation of the two sets.</li>
<li>'mcquitty': Similar to average, but uses a linkage function that is a variant of the WPGMA method (weighted pair group method with averaging).</li>
<li>'median': Uses the median of the distances between all observations of the two sets.</li>
<li>'centroid': Uses the centroid of the clusters.</li>
</ul>
</li>
<li><strong>Title</strong>: The title of the plot.</li>
</ul>
<h4 id="interpretation-of-analysis-results-6">Interpretation of analysis results </h4>
<p>Afterwards, the substrates from the two clusters are imported into the KSEA module of PhosMap to infer kinase activities. The results indicate that CDK1/2, MAPK1/3 and AKT1 are suppressed during BRAFi treatment.</p>
<h3 id="motif-enrichment-analysis">Motif enrichment analysis </h3>
<h4 id="function-15">Function </h4>
<p>PhosMap allowed for performing MEA on user defined phosphopeptides lists to provide clues for finding candidate kinases that are not present in the database.</p>
<h4 id="the-meaning-of-parameters">The meaning of parameters </h4>
<ol>
<li>‘Fasta type’ refers to the type of fasta file used as input for the analysis.</li>
<li>‘Selected row number for plotting motif logo’ is the number of rows to be selected for generating the motif logo plot.</li>
<li>‘Matched seqs threshold’ is the threshold for determining the minimum number of matched sequences required for a motif to be considered significant.</li>
<li>‘Scale’ is a parameter for scaling the data before generating the heatmap.</li>
<li>‘Distance metric’ is a parameter for heatmap generation that specifies the distance metric used for clustering rows.</li>
<li>‘Clustering method’ is a parameter for heatmap generation that specifies the clustering method used to cluster rows and columns.</li>
</ol>
<h4 id="how-to-get-analysis-results-14">How to get analysis results </h4>
<ol>
<li>Go to the ‘Motif Enrichment Analysis’ under ‘Analysis’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the ‘Analysis’ button.</li>
<li>The foreground dataframe mapped to motifs is shown on the right after running.</li>
<li>Select row number for plotting logo.</li>
<li>Click the first ‘Plot’ button, and the logo will appear on the right.</li>
<li>Modify the parameters below.</li>
<li>Click the second ‘Plot’ button.</li>
<li>The heatmap will appear on the right.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/motif1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/motif2.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/motif3.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-13">Parameter Selection Explanation </h4>
<ul>
<li><strong>species</strong>: Users select 'human', 'mouse' or 'rattus' to map peptide sequences against the human protein database.</li>
<li><strong>fasta type</strong>: Indicates the database format for protein sequences. Options are 'refseq' for NCBI's RefSeq database or 'uniprot' for the UniProt database. Selection should align with the user's sequence data and the ID type chosen to ensure consistency in mapping results.</li>
<li><strong>pvalue threshold</strong>: Sets the cutoff for statistical significance, typically at 0.05. Lower this threshold for stricter criteria.</li>
<li><strong>Matched Seqs Threshold</strong>: Determines the minimum number of sequences that must match a motif for it to be included in the heatmap. A higher threshold may focus the analysis on more prevalent motifs, while a lower threshold allows inclusion of less common motifs.</li>
<li><strong>Scale</strong>: Controls whether and how data normalization is applied. 'None' means no normalization; scaling can otherwise adjust the data to emphasize variability ('row') or comparability ('column').</li>
<li><strong>Distance Metric</strong>: Specifies how the similarity between data points is measured. 'Euclidean' measures direct distance, which could prioritize absolute differences in values.</li>
<li><strong>Clustering Method</strong>: The algorithm used to cluster rows or columns in the heatmap, which influences how the patterns of similarity are grouped.
<ul>
<li>'ward.D2': Minimizes the sum of squared differences within all clusters.</li>
<li>'ward.D': An alternative Ward's method.</li>
<li>'single': Uses the minimum of the distances between all observations of the two sets.</li>
<li>'complete': Uses the maximum distances between all observations of the two sets.</li>
<li>'average': Uses the average of the distances of each observation of the two sets.</li>
<li>'mcquitty': Similar to average, but uses a linkage function that is a variant of the WPGMA method (weighted pair group method with averaging).</li>
<li>'median': Uses the median of the distances between all observations of the two sets.</li>
<li>'centroid': Uses the centroid of the clusters.</li>
</ul>
</li>
<li><strong>Title</strong>: The title of the plot.</li>
</ul>
<h4 id="interpretation-of-analysis-results-7">Interpretation of analysis results </h4>
<p>The 3,649 identified phosphor-peptides as foreground sequences are used for MEA of PhosMap and the results further strengthen the evidence of CDK and MAPK pathway deactivation in BRAF mutant CRC cells in response to BRAFi treatment.</p>
<h3 id="survival-analysis">Survival analysis </h3>
<h4 id="function-16">Function </h4>
<p>This module is used to identify phosphorylation sites or kinases  associated with clinical outcomes of patients. Using kinases or phosphorylation locations files and patients’ survival information as input matrices, coxph function from survival R package was used to calculate the hazard ratio (HR) and P-value.</p>
<h4 id="how-to-get-analysis-results-15">How to get analysis results </h4>
<ol>
<li>Go to the ’Survival Analysis’ under ‘Analysis’ tab.</li>
<li>Modify the parameters according to your needs.</li>
<li>Click the ‘Analysis’ button.</li>
<li>The summary dataframe list will appear on the right.</li>
<li>Click the ‘Plot’ button.</li>
<li>The plot after running will appear on the right.</li>
<li>Click the download button to download the plot file.</li>
</ol>
<div style="text-align: center;">
  <img src="manual_picture2/analysis/surv1.png" style="max-width: 100%;">
  <img src="manual_picture2/analysis/surv2.png" style="max-width: 100%;">
</div>
<h4 id="parameter-selection-explanation-14">Parameter Selection Explanation </h4>
<ul>
<li><strong>pvalue adjust method</strong>: Selects the correction method for multiple comparisons. Available methods include:
<ul>
<li>'none': No adjustment for multiple comparisons.</li>
<li>'holm': Holm's sequential Bonferroni method.</li>
<li>'hochberg': Hochberg's step-up procedure.</li>
<li>'hommel': Hommel's correction.</li>
<li>'bonferroni': Bonferroni correction, very stringent, increases with the number of tests.</li>
<li>'BH': Benjamini-Hochberg method, controls the false discovery rate.</li>
<li>'BY': Benjamini-Yekutieli method, suitable for when tests are dependent.</li>
<li>'fdr': A general term for any method that controls the false discovery rate.</li>
</ul>
</li>
<li><strong>pvalue threshold</strong>: Sets the cutoff for statistical significance, typically at 0.05. Lower this threshold for stricter criteria.</li>
<li><strong>"high" colour</strong>: The color coding for higher values or significant results in the output. The choice of color can affect the interpretability of the visualization.</li>
<li><strong>"low" colour</strong>: The color coding for lower values or non-significant results in the output. Similar to the "high" color, the choice here can influence visual contrast and readability.</li>
</ul>
<h2 id="references">References </h2>
<ol>
<li>Feng, J., Ding, C., Qiu, N., Ni, X., Zhan, D., Liu, W., Xia, X., Li, P., Lu, B. and Zhao, Q. (2017) Firmiana: towards a one-stop proteomic cloud platform for data processing and analysis. Nature biotechnology, 35, 409-412.</li>
<li>Ressa, A., Bosdriesz, E., De Ligt, J., Mainardi, S., Maddalo, G., Prahallad, A., Jager, M., De La Fonteijne, L., Fitzpatrick, M. and Groten, S. (2018) A system-wide approach to monitor responses to synergistic BRAF and EGFR inhibition in colorectal cancer cells. Molecular &amp; Cellular Proteomics, 17, 1892-1908.</li>
</ol>

      </div>
      <div class="md-sidebar-toc">
<div class="md-toc">
<details style="padding:0;;padding-left:0px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#procedure" class="md-toc-link"><p>PROCEDURE</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#notice" class="md-toc-link">
            <p>Notice</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#introduction-of-example-data" class="md-toc-link">
            <p>Introduction of example data</p>

          </a></div><details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#preprocessing-for-maxquant-data" class="md-toc-link"><p>Preprocessing for Maxquant data</p>
</a>
          </summary>
        <div>
          <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#import-maxquant-data" class="md-toc-link"><p>Import MaxQuant data</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-import-your-maxquant-data" class="md-toc-link">
            <p>How to import your MaxQuant data</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#quality-control-and-merging" class="md-toc-link"><p>Quality control and merging</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#data-normalization" class="md-toc-link"><p>Data normalization</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-1" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-1" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-1" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#preprocessing-for-firmiana-data" class="md-toc-link"><p>Preprocessing for Firmiana data</p>
</a>
          </summary>
        <div>
          <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#import-firmiana-data" class="md-toc-link"><p>Import Firmiana data</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-import-your-firmiana-data" class="md-toc-link">
            <p>How to import your Firmiana data</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#parser" class="md-toc-link"><p>Parser</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-2" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-2" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#quality-control-and-merging-1" class="md-toc-link"><p>Quality control and merging</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-3" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-3" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-2" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-1" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#mapping-p-sites-to-protein" class="md-toc-link"><p>Mapping p-sites to protein</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-4" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-4" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-3" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-2" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#data-normalization-1" class="md-toc-link"><p>Data normalization</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-5" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-5" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-4" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#preprocessing-for-spectronaut-data" class="md-toc-link"><p>Preprocessing for Spectronaut data</p>
</a>
          </summary>
        <div>
          <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#import-spectronaut-data" class="md-toc-link"><p>Import Spectronaut data</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-import-your-spectronaut-data" class="md-toc-link">
            <p>How to import your Spectronaut data</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#parser--p-site-quality-control" class="md-toc-link"><p>Parser &amp; p-site Quality Control</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-6" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-6" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-5" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#normalizaiton--imputation--filtering" class="md-toc-link"><p>Normalizaiton &amp; Imputation &amp; Filtering</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-7" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-7" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-6" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#preprocessing-for-dia-nn-data" class="md-toc-link"><p>Preprocessing for Dia-NN data</p>
</a>
          </summary>
        <div>
          <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#import-dia-nn-data" class="md-toc-link"><p>Import Dia-NN data</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-import-your-dia-nn-data" class="md-toc-link">
            <p>How to import your Dia-NN data</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#parser--p-site-quality-control-1" class="md-toc-link"><p>Parser &amp; p-site Quality Control</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-8" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-8" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-7" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#normalizaiton--imputation--filtering-1" class="md-toc-link"><p>Normalizaiton &amp; Imputation &amp; Filtering</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-9" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-9" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-8" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#analysis-and-visualization" class="md-toc-link"><p>Analysis and visualization</p>
</a>
          </summary>
        <div>
          <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#upload-data" class="md-toc-link"><p>Upload data</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-10" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-upload-your-data" class="md-toc-link">
            <p>How to upload your data</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#dimension-reduction-analysis" class="md-toc-link"><p>Dimension reduction analysis</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-11" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#the-meaning-of-the-parameters" class="md-toc-link">
            <p>The meaning of the parameters</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-10" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-9" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-3" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#differential-expression-analysis" class="md-toc-link"><p>Differential expression analysis</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-12" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#the-meaning-of-the-parameters-1" class="md-toc-link">
            <p>The meaning of the parameters</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-11" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-10" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-4" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#time-course-analysis" class="md-toc-link"><p>Time Course Analysis</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-13" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#the-meaning-of-the-parameters-2" class="md-toc-link">
            <p>The meaning of the parameters</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-12" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-5" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-11" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#kinase-activity-prediction-ksea" class="md-toc-link"><p>Kinase activity prediction (KSEA)</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-14" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#the-meaning-of-the-parameters-3" class="md-toc-link">
            <p>The meaning of the parameters</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-13" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-12" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-6" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#motif-enrichment-analysis" class="md-toc-link"><p>Motif enrichment analysis</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-15" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#the-meaning-of-parameters" class="md-toc-link">
            <p>The meaning of parameters</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-14" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-13" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#interpretation-of-analysis-results-7" class="md-toc-link">
            <p>Interpretation of analysis results</p>

          </a></div>
        </div>
      </details>
    <details style="padding:0;;padding-left:24px;" open="">
        <summary class="md-toc-link-wrapper">
          <a href="#survival-analysis" class="md-toc-link"><p>Survival analysis</p>
</a>
          </summary>
        <div>
          <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#function-16" class="md-toc-link">
            <p>Function</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#how-to-get-analysis-results-15" class="md-toc-link">
            <p>How to get analysis results</p>

          </a></div><div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#parameter-selection-explanation-14" class="md-toc-link">
            <p>Parameter Selection Explanation</p>

          </a></div>
        </div>
      </details>
    
        </div>
      </details>
    <div class="md-toc-link-wrapper" style="padding:0;;display:list-item;list-style:square;margin-left:42px">
          <a href="#references" class="md-toc-link">
            <p>References</p>

          </a></div>
        </div>
      </details>
    
</div>
</div>
      <a id="sidebar-toc-btn">≡</a>
    
    
    
    
    
    
<script>

var sidebarTOCBtn = document.getElementById('sidebar-toc-btn')
sidebarTOCBtn.addEventListener('click', function(event) {
  event.stopPropagation()
  if (document.body.hasAttribute('html-show-sidebar-toc')) {
    document.body.removeAttribute('html-show-sidebar-toc')
  } else {
    document.body.setAttribute('html-show-sidebar-toc', true)
  }
})
</script>
      
  
    </body></html>